File: RHOMBUS/rhombus.java
class Main
{
    public static void main(String[] args)
    {
        // number of rows
        final int n = 5;
 
        for (int i = 1; i <= n; i++)
        {
            // print space
            for (int j = i; j < n; j++) {
                System.out.print(' ');
            }
 
            // Print `n` stars
            for (int j = 1; j <= n; j++) {
                System.out.print('*');
            }
 
            // move to the next row
            System.out.print(System.lineSeparator());
        }
    }
}

File: RHOMBUS/rhombus.c
#include <stdio.h>
int main()
{
    // number of rows
    int n = 5;
    int i, j;
 
    for (i = 1; i <= n; i++)
    {
        // print space
        for (j = i; j < n; j++) {
            printf(" ");
        }
 
        // print `n` stars
        for (j = 1; j <= n; j++) {
            printf("*");
        }
 
        // move to the next row
        printf("\n");
    }
 
    return 0;
}
